pingcap/tidb
# Telemetry: Add slow query statistic bucket into telemetry data

### What problem does this PR solve?

Issue Number: close #25005

Problem Summary:

### What is changed and how it works?
Add slow query statistic bucket into telemetry data
Proposal: [xxx](url) 

What's Changed:

How it Works:
Work with telemetry framework.
### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
1. Setup TiDB
2. Create a query which process time is over 300 ms, suggest join.
3. Use: admin show telemetry check whether the slow query buckets displayed.

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 
- Add slow query statistic buckets into telemetry data.
